NewHaven/src/GBMap.cpp

```diff
@@ -246,3 +248,59 @@ Building* GBMap::drawBuildingFromBoard(int position) {
         }
     return nullptr;
 }
+
+void GBMap::assignDefaultTiles() {
+    /*
+     * Create 4 tiles
+     */
+    ResourceTypes *type1{new ResourceTypes[4]
+    {ResourceTypes::STONE, ResourceTypes ::SHEEP, ResourceTypes::WOOD, ResourceTypes::WOOD}};
+    ResourceTypes *type2{new ResourceTypes[4]
+    {ResourceTypes::WHEAT, ResourceTypes ::SHEEP, ResourceTypes::WHEAT, ResourceTypes::WOOD}};
+    ResourceTypes *type3{new ResourceTypes[4]
+    {ResourceTypes::SHEEP, ResourceTypes ::STONE, ResourceTypes::SHEEP, ResourceTypes::WHEAT}};
+    ResourceTypes *type4{new ResourceTypes[4]
+    {ResourceTypes::STONE, ResourceTypes ::STONE, ResourceTypes::WOOD, ResourceTypes::WHEAT}};
+    HarvestTile tl{type1};
+    HarvestTile tr{type2};
+    HarvestTile bl{type3};
+    HarvestTile br{type4};

```

Comment:
You can create the harvest tiles directly:
`HarvestTile tile{new ResourceTypes[4] {ResourceTypes::STONE, ResourceTypes ::SHEEP, ResourceTypes::WOOD, ResourceTypes::WOOD}};`
It doesn't make a difference performance wise, it's a matter of preference.

Reply:
A driver didn't seem necessary. I just manually place tiles using the same function we have always used. If an error occurs it would be extremely worrisome. However, none occur, if it did you would get an error from the Part1 driver since this function is called in the constructor of the game board.